lib/internal/cluster/child.js

```diff
@@ -177,7 +194,7 @@ function rr(message, { indexesKey, index }, cb) {
     // the primary.
     if (key === undefined)
       return;
-
+    unref();
     send({ act: 'close', key });
```

Comment:
This message is going to get lost, right? After `unref()` there's nothing keeping the event loop alive anymore if it's the last handle.

Reply:
Yes, i have updated the code to `send({ act: 'close', key, needAck: true }, () => unref());` and parent process will send a ack when receive a close request.

Reply:
And it seems parent process will remove the handle(handles.delete(key)) when the worker process exits, i think it is ok if the close message get lost ?

Reply:
Test failures look related. Perhaps `'close'` getting lost is an issue that was already present.

You're right that the handles are going to get cleaned up anyway so I guess it's okay to call `send()` without a callback but can you add a comment explaining that?

Reply:
Done !